nss-evening-cohort-14/michelin-star-restaurant-dashboard-michelin-star-restaurant-dashboard-1
# DELETE: Reservations

## Description
Added in delete buttons and functionality for reservations

## Related Issue
#29 

## Motivation and Context
Needed the ability to delete reservations from list

## How Can This Be Tested?
Go to reservations in the nav as a logged in user and hit the delete button to see a reservation delete

## Types of changes
